locales-src/newtab.ftl

```diff
@@ -0,0 +1,159 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### Firefox Home / New Tab strings for about:home / about:newtab / about:welcome
+
+newtab-page-title = New Tab
+newtab-settings-button =
+    .title = Customize your New Tab page
+
+## Search box component
+
+# "Search" is a verb/action
+newtab-search-box-search-button =
+    .title = Search
+    .aria-label = Search
+
+newtab-search-box-search-the-web-text = Search the Web
+newtab-search-box-search-the-web-input =
+    .placeholder = Search the Web
+    .title = Search the Web
+    .aria-label = Search the Web
+
+## Top Sites - General form dialog
+
+newtab-topsites-add-header = New Top Site
+newtab-topsites-edit-header = Edit Top Site
+newtab-topsites-title-label = Title
+newtab-topsites-title-input =
+    .placeholder = Enter a title
+
+newtab-topsites-url-label = URL
+newtab-topsites-url-input =
+    .placeholder = Type or paste a URL
+newtab-topsites-url-validation = Valid URL required
+
+newtab-topsites-image-url-label = Custom Image URL
+newtab-topsites-use-image-link = Use a custom image…
+newtab-topsites-image-validation = Image failed to load. Try a different URL.
+
+## Top Sites - General form dialog buttons. These are verbs/actions
+newtab-topsites-cancel-button = Cancel
+newtab-topsites-delete-history-button = Delete from History
+newtab-topsites-save-button = Save
+newtab-topsites-preview-button = Preview
+newtab-topsites-add-button = Add
+
+## Top Sites - Delete history confirmation dialog. newtab-confirm-delete-history-p1 appears in the same dialog
+## as newtab-confirm-delete-history-p2.
+newtab-confirm-delete-history-p1 = Are you sure you want to delete every instance of this page from your history?
+# Note: "This action" refers to deleting a page from history.
+newtab-confirm-delete-history-p2 = This action cannot be undone.
+
+newtab-topsites-add-search-engine = Add Search Engine
```

Comment:
@Pike  By group comment you mean comments with '##' which will show up in pontoon for all ongoing entries till next group comment is encountered. Reviewed newtab.ftl for comments and looks good except above string which needs its own group comment

Reply:
@JeaneC As discussed, lets move this string with top sites dialog and have a new string for 'add search engine' as menu option